content/blog/2019/announcing-1.1/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Announcing Istio 1.1
+description: Istio 1.1 release announcement.
+publishdate: 2019-03-14
+attribution: The Istio Team
+weight: 77
+---
+We are pleased to announce the release of Istio 1.1.
+
+Since we released 1.0 back in July, we’ve done a lot of work to help people
+get into production. Not surprisingly, we had to do some patch releases (6 so
+far!), but we’ve also been hard at work adding new features to the product.
+
+The theme for 1.1 is Enterprise Ready. We’ve been very pleased to see more
+and more companies using Istio in production, but as some larger companies
+tried to adopt Istio they hit some limits.
+
+One of our prime areas of focus has been performance and scalability. As people
+moved into production with larger clusters running more services at higher
+volume, they hit some scaling and performance issues. The sidecars took too
+many resources and added too much latency. The control plane (especially Pilot)
+was overly resource hungry.
+
+We’ve done a lot of work to make both the data plane and the control plane
+more efficient. In our 1.1 performance testing, we’re now seeing a sidecar
+that typically takes half of a vCPU to process 1000 rps. A single Pilot
+instance is capable of handling 1000 services (and 2000 total pods) while
+consuming 1.5 vCPU and 2GB of RAM. The sidecar adds 5ms at the 50th
+percentile and 10ms at 99th percentile (enforcing policy will add latency).
+
+We’ve done work around namespace isolation as well. This lets you use
+Kubernetes namespaces to enforce boundaries of control, and ensures that your
+teams cannot interfere with each other.
+
+We have also improved the multicluster capabilities and usability. We
+listened to the community and improved defaults for traffic control and policy.
+We introduced a new component called Galley. Galley validates that sweet, sweet
+YAML, reducing the chance of configuration errors. Galley will also be
+instrumental in multicluster setups, gathering service discovery information
+from each Kubernetes cluster. We are also supporting additional multicluster
+topologies including single control plane and multiple synchronized control
+planes without requiring a flat network.
+
+There is lots more -- see release notes here.
```

Comment:
[release notes](/docs/about/notes/1.1/)

Reply:
I linked it to /notes/about/1.1/ which seems appropriate.